crosscheckFingerprints.wdl

```diff
@@ -56,13 +56,11 @@ task runCrosscheckFingerprints {
         String haplotypeMap
         String outputPrefix
         String crosscheckBy = "READGROUP"
-        Int picardMaxMemMb = 3000
         Int exitCodeWhenMismatch = 0
         Int exitCodeWhenNoValidChecks = 0
         Float lodThreshold = 0.0
         String validationStringency = "SILENT"
-        String modules = "picard/2.21.2 crosscheckfingerprints-haplotype-map/20210201"
-        String? additionalParameters
+        String modules = "gatk/4.2.0.0 crosscheckfingerprints-haplotype-map/20210201"
```

Comment:
the fingerprint generation uses a more recent version of the haplotype-map
crosscheckfingerprints-haplotype-map/20230324

https://github.com/oicr-gsi/crosscheckFingerprintsCollector/blob/384a27a5596e35e820a5124b6fd7b5d7465cfc26/crosscheckFingerprintsCollector.wdl#L67

I'm not sure what the impact here would be but it would be better to keep these consistent

Reply:
I checked the module file and it's the same files + some new bedfiles for parallization (not used here). I will update to keep everything in sync.